### PR TITLE
Taged all version with release vesion

### PR DESCRIPTION
**Pull request by evalsocket**
Added workflow in release for tagging all component with flyte release 

Fix: https://github.com/flyteorg/flyte/issues/1787

Workflow Test: https://github.com/evalsocket/flyte/actions/runs/1539015766

Helm Charts: https://github.com/evalsocket/flyte/blob/gh-pages/flyte-sandbox-v4.0.0.tgz & https://github.com/evalsocket/flyte/blob/gh-pages/flyte-sandbox-v4.0.0.tgz

Docker Image: https://github.com/evalsocket?tab=packages&q=release

### CODE REVIEW

File: script/prepare_artifacts.sh [hunk 1]
```diff
@@ -21,3 +21,15 @@ grep -rlZ "version:[^P]*# VERSION" ./charts/flyte-core/Chart.yaml | xargs -0 sed
 sed "s/v0.1.10/${VERSION}/g" ./charts/flyte-core/README.md  > temp.txt && mv temp.txt ./charts/flyte-core/README.md
 
 helm dep update ./charts/flyte
+
```
[the remark this answers is not in the document]
Comment (evalsocket) [reply script/prepare_artifacts.sh:24]: yes but here we use it for preparing the helm charts in CI. Going forward we will remove the other script that is used by flyte-bot for creating PR with latest component.

File: charts/flyte-core/values.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ flyteadmin:
   replicaCount: 1
   image:
     # -- Docker image for Flyteadmin deployment
-    repository: cr.flyte.org/flyteorg/flyteadmin
+    repository: cr.flyte.org/flyteorg/flyteadmin # FLYTEADMIN_IMAGE
```
Comment (EngHabu) [inline charts/flyte-core/values.yaml:18]: Shouldn't these change to be
```suggestion
    repository: cr.flyte.org/flyteorg/flyteadmin-release # FLYTEADMIN_IMAGE
```
And the same for the others?

Comment (evalsocket) [reply charts/flyte-core/values.yaml:18]: No that is not required, We only prepare them for release.

Comment (evalsocket) [reply charts/flyte-core/values.yaml:18]: @EngHabu it's ready now

**Pull request merged**